example/lib/nodes.dart

```diff
@@ -14,17 +14,11 @@ FlPortStyle outputDataPortStyle(state) => FlPortStyle(
       color: state.isHovered ? Colors.limeAccent : Colors.deepOrange,
       shape: FlPortShape.circle,
       linkStyleBuilder: (state) => FlLinkStyle(
-        useGradient: false,
         color: state.isSelected
             ? Colors.orangeAccent
             : state.isHovered
                 ? Colors.limeAccent
                 : Colors.deepOrange,
-        gradient: const LinearGradient(
-          colors: [Colors.deepOrange, Colors.purple],
-          begin: Alignment.centerLeft,
-          end: Alignment.centerRight,
-        ),
```

Comment:
I removed the `gradient` values there cause you're not really supposed to specify both with the second commit.

However, from my understanding this is supposed to serve as a demo of the fact links can have gradients, correct? So to keep that effect, I suggest either:
- commenting the `gradient` lines instead of removing them
- using gradients for only one of the ports
- having a file-local bool that the style builders use to decide whether it should give links gradients or solid colors